### PR TITLE
Front: Adn/add grid margin

### PR DESCRIPTION
**Pull request by alexandredamiron**
Ok, it may seem complicate for just two input in stdcm. But it also begins a isolation & composition of Allowance Component

### CODE REVIEW

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js [hunk 1]
```diff
@@ -162,56 +191,88 @@ export default function StandardAllowanceDefault(props) {
...
-          {t('apply')}
-        </button>
-        <button
-          type="button"
-          onClick={() => delStandard(value)}
-          className={`btn btn-danger btn-sm ${value.value === 0 ? 'disabled' : null}`}
-        >
-          <FaTrash />
-        </button>
+      <div className="col-md-2 text-normal">{title || t('sandardAllowancesWholePath')}</div>
```
Comment (nicolaswurtz) [inline front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: Useless `row > col` here, `md-2` is too small in reduced screenwidth, prefer put text as a label.
Beware of left & right margins not equals.

Just removing `row w-100` in parent and `col-md-2` is enough as proposed in screen capture below :
image

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js [hunk 2]
```diff
@@ -162,56 +191,88 @@ export default function StandardAllowanceDefault(props) {
...
+
+      <div className="col">
+      <div className="row">
+          {options.setDistribution && (
+            <React.Fragment>
+            <div className="col-md-2 text-normal">{t('Valeur par défault')}</div>
```
Comment (nicolaswurtz) [inline front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: When using locale, you have to use english and concise text in camelCase.
`t('defaultValue')` for example.

**Review by nicolaswurtz — CHANGES_REQUESTED** [review]

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js (hunk 1 above)
Comment (alexandredamiron) [reply front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: conditionnal layout in pace

File: front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js (hunk 2 above)
Comment (alexandredamiron) [reply front/src/applications/osrd/components/Simulation/Allowances/StandardAllowanceDefault.js]: done

**Review by nicolaswurtz — APPROVED** [review]

**Pull request merged**